docs/sources/setup/install/helm/deployment-guides/azure.md

```diff
@@ -0,0 +1,588 @@
+
+In this guide, we deploy Loki using `Standard_E2ds_v5` instances. This is to make sure we remain within the free tier limits for Azure. Which allows us to deploy up to 10 vCPUs within a region. We recommend for large production workloads to scale these nodes up to `Standard_D4_v5`.
+{{< /admonition >}}
+
+The minimum requirements for deploying Loki on AKS are: 
+
+- Kubernetes version `1.30` or above.
+- `3` nodes for the AKS cluster.
+- Instance type depends on your workload. A good starting point for a production cluster in the free tier is `Standard_E2ds_v5` instances and for large production workloads `Standard_D4_v5` instances.
+
+Here is how to create an AKS cluster with the minimum requirements:
+
+```bash
+az aks create \
+  --resource-group <MY_RESOURCE_GROUP_NAME> \
+  --name <MY_AKS_CLUSTER_NAME> \
+  --node-count 3 \
+  --node-vm-size Standard_E2ds_v5 \
+  --generate-ssh-keys \
+  --enable-workload-identity \
+  --enable-oidc-issuer
+```
+
+Note in the above command we have enabled workload identity and OIDC issuer. This is required for the Loki service account to authenticate with Azure AD. If you have already created an AKS cluster, you can enable these features using the following command:
+
+```bash
+az aks update \
+  --resource-group <MY_RESOURCE_GROUP_NAME> \
+  --name <MY_AKS_CLUSTER_NAME> \
+  --enable-workload-identity \
+  --enable-oidc-issuer
+```
+
+The Azure CLI also allows you to bind the AKS cluster to kubectl. You can do this by running the following command:
+
+```bash
+az aks get-credentials --resource-group <MY_RESOURCE_GROUP_NAME> --name <MY_AKS_CLUSTER_NAME>
+```
+
+## Configuring Azure Blob Storage
+
+{{< admonition type="tip" >}}
+ **Consider** using unique bucket names rather than;  `chunk`, `ruler` and `admin`. Although Azure Blog Storage is not directly affected by this [security update](https://grafana.com/blog/2024/06/27/grafana-security-update-grafana-loki-and-unintended-data-write-attempts-to-amazon-s3-buckets/) it is best practise to use unique container names (buckets).
+{{< /admonition >}}
+
+Before deploying Loki, you need to create two Azure storage containers; one to store logs (chunks), the second to store alert rules. You can create the containers using the Azure CLI. Containers must exist inside a storage account: 
+
+{{< admonition type="note" >}}
+GEL customers will require a third container to store the admin data. This container is not required for OSS users.
+{{< /admonition >}}
+
+1. Create a storage account:
+
+    ```bash
+    az storage account create \
+    --name <NAME> \
+    --location <REGION> \
+    --sku Standard_ZRS \
+    --encryption-services blob \
+    --resource-group <MY_RESOURCE_GROUP_NAME>
+    ```
+    **Replace the placeholders with your desired values.** We assume you have a resource group created.
```

Comment:
Since we created a resource group up at line 67, I don't think you need this.
```suggestion
    **Replace the placeholders with your desired values.**
```